opencog/persist/zmq/events/AtomSpacePublisherModule.cc

```diff
@@ -365,6 +367,24 @@ Object AtomSpacePublisherModule::tvToJSON(TruthValuePtr tvp)
             break;
         }
 
+        case PROBABILISTIC_TRUTH_VALUE: {
+            json.push_back(Pair("type", "probabilistic"));
+            jsonDetails.push_back(Pair("strength", tvp->getMean()));
+            jsonDetails.push_back(Pair("count", tvp->getCount()));
+            jsonDetails.push_back(Pair("confidence", tvp->getConfidence()));
+            json.push_back(Pair("details", jsonDetails));
+            break;
+        }
+
+        case FUZZY_TRUTH_VALUE: {
+            json.push_back(Pair("type", "fuzzy"));
+            jsonDetails.push_back(Pair("strength", tvp->getMean()));
+            jsonDetails.push_back(Pair("count", tvp->getCount()));
+            jsonDetails.push_back(Pair("confidence", tvp->getConfidence()));
```

Comment:
P.S.  I don't know why STV and FTV's constructor in C++ takes count, when the construction from Scheme takes confidence value.  Even the output is in confidence value.